dear_petition/petition/utils.py

```diff
@@ -0,0 +1,15 @@
+from django.utils.timezone import make_aware
+import pytz
+from datetime import datetime, date
+
+
+def dt_obj_to_date(dt_obj):
+    """converts a datetime obj to a date"""
+    return dt_obj.date()
+
+
+def make_datetime_aware(dt_str):
```

Comment:
`line 127` `models.py`.  I think we are passing `None` instead of `""` if there is not a `offense date`. I didn't update the migration though. I updated the migration to use `None` instead.  Will add a test.